distribution/src/main/resources/bin/plugin

```diff
@@ -59,6 +53,12 @@ if [ -n "$ES_ENV_FILE" ]; then
   fi
 fi
 
+# CONF_FILE setting was removed
+if [ ! -z "$CONF_FILE" ]; then
+    echo "CONF_FILE setting is no longer supported. elasticsearch.yml must be placed in config folder and cannot be renamed."
```

Comment:
in config folder -> in the config directory

Note I suggest directory here instead of folder, since this is a nix script